packages/opencensus-core/src/stats/aggregation/types.ts

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright 2018, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Measurement, Tags} from '../types';
+
+/**
+ * Informs the type of the aggregation. It can be: count, sum, lastValue or
+ * distribution.
+ */
+export const enum AggregationType {
+  count = 0,
```

Comment:
Could you make enum fields all caps?

Reply:
Makes sense to me. Updated here and in other enums